tests/upsert_validator/test_validator_election.py

```diff
@@ -95,3 +94,74 @@ def test_upsert_validator_invalid_election(b_mock, new_validator, node_key):
 
     with pytest.raises(UnequalValidatorSet):
         tx_election.validate(b_mock)
+
+
+def test_get_status_ongoing(b, ongoing_election, new_validator):
+    status = ValidatorElection.ONGOING
+    resp = ongoing_election.get_status(b)
+    assert resp == status
+
+
+def test_get_status_concluded(b, concluded_election, new_validator):
+    status = ValidatorElection.CONCLUDED
+    resp = concluded_election.get_status(b)
+    assert resp == status
+
+
+def test_get_status_inconclusive(b, inconclusive_election, new_validator):
+    def custom_mock_get_validators(height):
+        if height >= 3:
+            return [{'pub_key': {'data': 'zL/DasvKulXZzhSNFwx4cLRXKkSM9GPK7Y0nZ4FEylM=',
+                                 'type': 'AC26791624DE60'},
+                     'voting_power': 15},
+                    {'pub_key': {'data': 'GIijU7GBcVyiVUcB0GwWZbxCxdk2xV6pxdvL24s/AqM=',
+                                 'type': 'AC26791624DE60'},
+                     'voting_power': 7},
+                    {'pub_key': {'data': 'JbfwrLvCVIwOPm8tj8936ki7IYbmGHjPiKb6nAZegRA=',
+                                 'type': 'AC26791624DE60'},
+                     'voting_power': 10},
+                    {'pub_key': {'data': 'PecJ58SaNRsWJZodDmqjpCWqG6btdwXFHLyE40RYlYM=',
+                                 'type': 'AC26791624DE60'},
+                     'voting_power': 8}]
+        else:
+            return [{'pub_key': {'data': 'zL/DasvKulXZzhSNFwx4cLRXKkSM9GPK7Y0nZ4FEylM=',
+                                 'type': 'AC26791624DE60'},
+                     'voting_power': 9},
+                    {'pub_key': {'data': 'GIijU7GBcVyiVUcB0GwWZbxCxdk2xV6pxdvL24s/AqM=',
+                                 'type': 'AC26791624DE60'},
+                     'voting_power': 7},
+                    {'pub_key': {'data': 'JbfwrLvCVIwOPm8tj8936ki7IYbmGHjPiKb6nAZegRA=',
+                                 'type': 'AC26791624DE60'},
+                     'voting_power': 10},
+                    {'pub_key': {'data': 'PecJ58SaNRsWJZodDmqjpCWqG6btdwXFHLyE40RYlYM=',
+                                 'type': 'AC26791624DE60'},
+                     'voting_power': 8}]
+
+    b.get_validators = custom_mock_get_validators
+    status = ValidatorElection.INCONCLUSIVE
+    resp = inconclusive_election.get_status(b)
+    assert resp == status
+
+
+def test_upsert_validator_show(caplog, ongoing_election, b):
+    from bigchaindb.commands.bigchaindb import run_upsert_validator_show
+
+    election_id = ongoing_election.id
+    public_key = public_key_to_base64(ongoing_election.asset['data']['public_key'])
+    power = ongoing_election.asset['data']['power']
+    node_id = ongoing_election.asset['data']['node_id']
+    status = ValidatorElection.ONGOING
+
+    show_args = Namespace(action='show',
+                          election_id=election_id)
+
+    # TODO: Turn this back on once tests/commands/test_utils.test_configure_bigchaindb_logging.py is fixed
```

Comment:
I think, it's a good test already. We do not gain much by also testing that `msg` is also sent to the logger.

Reply:
See my above response about the `return` value. I don't think we actually need it, so the commented out code is there so that we can easily align the test once that goes away.